BackEnd/search/serializers.py

```diff
@@ -9,12 +10,33 @@
 from utils.regions_ukr_names import get_regions_ukr_names_as_string
 
 
+class ProfileImageField(serializers.Field):
+
+    def to_representation(self, value):
+        if value.is_deleted == False:
+            return {
+                "uuid": value.uuid,
+                "path": self.context["request"].build_absolute_uri(
+                    value.image_path.url
+                ),
+            }
+        return None
+
+    def to_internal_value(self, data):
+        profile_image = ProfileImage.objects.filter(
+            uuid=data, is_deleted=False
+        ).first()
+        return profile_image
```

Comment:
Again, you can remove some redundant code here.

Reply:
Done